deploy/pipeline-nightly-update.yaml

```diff
@@ -11,14 +11,22 @@ spec:
     name: cluster-credentials
     type: string
     default: openshift-pipelines-credentials
-  - description: Istio deployment. Only these values 'sail', 'ossm', 'ossm3'
+  - description: Istio deployment. Only 'ossm3' or '' allowed; Leave empty if installing on ocp4.19+
     name: istio-provider
     type: string
     default: ossm3
   - description: Kuadrant operator name. 'kuadrant-operator' or 'rhcl-operator'
     name: operator-name
     type: string
     default: kuadrant-operator
+  - description: GatewayAPI CRD version. v1.0.0, v1.1.0, v1.2.1 available; Leave empty if installing on ocp4.19+
+    name: gateway-crd
+    type: string
+    default: v1.2.1
+  - description: Keycloak subscription channel
+    name: keycloak-channel
+    type: string
+    default: stable-v26
```

Comment:
Are you sure that changing channel is sufficient and no need to change the startingCSV as well? I can see you say that it will be ignored but just double checking: https://github.com/azgabur/kuadrant-helm-install/blob/main/values.yaml#L57

Reply:
With my testing changing just the channel is enough to install the version v22 of Keycloak on ocp419. I think after GA the v26 will be available on ocp419l but if not, just changing the channel should be enough to prevent errors. 
The startingCSV does not have automatic installPlan approval added yet so using it in automated pipeline is not advised